hazelcast/src/main/java/com/hazelcast/sql/impl/client/ReadOptimizedLruCache.java

```diff
@@ -22,6 +22,7 @@
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicLongFieldUpdater;
+import java.util.function.Function;
 
 /**
  * Implementation of an LRU cache optimized for read-heavy use cases.
```

Comment:
`ReadOptimizedLruCache` should be moved to `com.hazelcast.internal.util.collection`. This can be done is separate commit.

Reply:
[`e1c3fd70..b132d684`][1]

[1]: https://github.com/hazelcast/hazelcast/compare/e1c3fd70d93fb0b4956297b6031479d37166855f..b132d684268595d3468902788ffb8a7e401c6df6